src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,55 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = null
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++
+    }, 10)
+
+    if (printTimeCallback) {
```

Comment:
Hay que meter esto en el setInterval anterior!! 
debajo de this.currentTime ++
if (printTimeCallback)printTimeCallback()